mkosi.md

```diff
@@ -1329,6 +1335,70 @@ turn-around times for complete image builds are minimal, as only
 changed source files need to be recompiled: an OS image rebuilt will
 be almost as quick to build the source tree only.
 
+# PRESETS
+
+A preset is just like a regular configuration, except that mkosi can
```

Comment:
Either "like a regular configuration file" or "like regular configuration" (configuration is uncountable).